Documentation/network/servicemesh/gateway-api/host-network-mode.rst

```diff
@@ -0,0 +1,103 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+Host network mode
+#################
+
+.. note::
+    Supported since Cilium 1.16+
+
+This mode allows you to expose the Cilium Gateway API Gateway directly on the
+host network.
+This is useful in some edge cases where one doesn't want, or can't, use
+LoadBalancer type services (dev environments, cluster-external loadbalancer,
+...).
```

Comment:
Two nits here:
1. Let's remove the parenthesis, as these are good examples to state directly in the sentence.
2. This sentence uses "one" to refer to the reader, but the previous sentence uses "you".

```suggestion
This is useful in edge cases where a LoadBalancer Service is unavailable, such
as in development environments or environments with cluster-external loadbalancers.
```